webfront/tests/test_sets.py

```diff
@@ -817,9 +705,7 @@ def test_can_filter_set_counter_with_structure_db(self):
         urls = ["/api/set/structure/pdb"]
         for url in urls:
             response = self.client.get(url)
-            self.assertEqual(
-                response.status_code, status.HTTP_200_OK, "URL : [{}]".format(url)
-            )
+            self.assertEqual(response.status_code, status.HTTP_200_OK, f"URL : [{url}]")
             self.assertIsInstance(response.data, dict)
             if "kegg" in response.data["sets"]:
                 self.assertIn(
```

Comment:
This block can be removed (we do not share KEGG data anymore, and they were never _sets_.